packages/performance-profiler-plugin/webpack.config.js

```diff
@@ -1,4 +1,11 @@
 const merge = require('webpack-merge');
 const webpackConfigBase = require('../webpack.config.base');
 
-module.exports = merge(webpackConfigBase);
+const cfg = {
+  devtool: 'eval',
```

Comment:
Is the eval setting necessary for the fix?

Reply:
Yes, I didn't manage to get it working without it.